docs/modules/ROOT/pages/how-to/zone-setup.adoc

```diff
@@ -117,3 +117,62 @@ parameters:
 
 . Compile and push the cluster catalog
 . Wait for Argo CD to sync the config
+
+== Configure metering metrics collection
+
+We collect a small set of metrics which are used for metering from each zone.
+This is done by deploying a https://thanos.io/tip/components/receive.md/[Thanos Receiver] on each zone and configure the zone's cluster monitoring to send selected metrics to the Thanos Receiver via Prometheus's remote write feature.
+
+. Create an S3 bucket which can be accessed using the same credentials as the cluster's registry bucket.
+Use name `${CLUSTER_ID}-appuio-metering` for the bucket.
+
+. Add configuration
```

Comment:
Is there a reason to not add this to the zone configuration?

Reply:
It is part of the zone configuration, but this how-to is the generic one which doesn't assume that you've got access to the VSHN global defaults (as far as I understand).